### DISCUSSION TITLE
GOBEL Test

### DISCUSSION THREAD

**Discussion by Manos1966** [skippermeister/OpenDTU-OnBattery#4] [General]
Hi @skippermeister 
dies zur Info:
ich hatte heute die Gelegenheit, die RS485 Schnittstelle an einer GOBEL GP-SR1-RN150 Batterie zu testen.
Es hat leider nicht funktioniert.
Erstes Problem war (natuerlich) die Batterie Position 2, die Pylontechs haben.
Wir haben also die GOBEL Batterie als 2 gestellt.

Wir haben die DEYE, PylonV2.9, Pylon V3.5 Konfiguration probiert
![GOBEL RS45 modes](https://github.com/skippermeister/OpenDTU-OnBattery/assets/129660360/36dd3cb8-a5b1-4adf-8be5-9ffd3d51bd09)

Trotzdem kein Erfolg.
![RS485 Buffer Overflow](https://github.com/skippermeister/OpenDTU-OnBattery/assets/129660360/bf7eb202-e767-4b35-bae5-0d0b3a03195a)

**Comment by skippermeister** [comment]
@Manos1966 
Meine nächste Erweiterung wird sein die Gobel RN und Gobel PC2000 Batterien per RS485 zu unterstützen. 
Die Protokoll Beschreibungen habe ich gefunden. Sie unterscheiden sich geringfügig von den Pylontech Batterien. Sollte aber kein Problem sein diese zu implementieren.
